recipes/eventio/meta.yaml

```diff
@@ -22,15 +17,15 @@ requirements:
   build:
     - {{ compiler('c') }}
   host:
-    - python {{ python }}
+    - python
     - pip
-    - numpy {{ numpy }}
+    - numpy
     - cython
     - pytest-runner
     - corsikaio 0.2.*
   run:
-    - python {{ python }}
-    - numpy {{ numpy }}
+    - python
+    - numpy
```

Comment:
If this builds against numpy please take a look https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Reply:
Thanks! It does indeed.

If we only require numpy >= 1.17 (we don't strictly require 1.17 here but only wish to support >= 1.17), should I include that or not?